conf/ocsci/fips.yaml

```diff
@@ -0,0 +1,3 @@
+---
+ENV_DATA:
+  fips: 'true'
```

Comment:
```suggestion
  fips: true
```
use boolean value and not string

Reply:
this is what I tried before and ended up the upper case with fips: True 
See the previous commit